src/modules/IscsiClientLib.rb

```diff
@@ -1711,6 +1730,8 @@ def GetDiscoveryCmd(ip, port, use_fw: false, only_new: false)
     publish :function => :GetOffloadModules, :type => "list <string> ()"
     publish :function => :LoadOffloadModules, :type => "list <string> ()"
     publish :function => :GetDiscoveryCmd, :type => "string (string, string, map)"
+    publish :function => :getCurrentNodeValues, :type => "map <string, any>) ()"
+    publish :function => :isiBFT, :type => "boolean (map <string, any>)"
```

Comment:
Um, does it actually work from non-Ruby code? The method name is actually `isiBFT?`. I'm not sure if ruby-bindings do some tricks here or not...

BTW the usual Ruby style is to not use the `is` prefix, `?` suffix is sufficient.